### PR TITLE
Closes #11655: expose HistoryVisitInfo.isRemote; update A-S to 91.0.1

### LINKED ISSUE #11655: Expose HistoryVisitInfo.isRemote
Relates to [..application-services/issue/4779](https://github.com/mozilla/application-services/issues/4779).

Basically, fenix wants to display history that was synced from other devices and that was created locally – separately. app-services added a field to HistoryVisitInfo, we should pass it to the consumer.

### PR DESCRIPTION
**Pull request by mavduevskiy**
Fixes [#11655](https://github.com/mozilla-mobile/android-components/issues/11655).

Looks like there is no need for tests, as it is just passing the value down.

It requires app-services to be updated to the latest version, I tested on current main: isRemote flag is there.

### CODE REVIEW

File: components/concept/storage/src/main/java/mozilla/components/concept/storage/HistoryStorage.kt [hunk 1]
```diff
@@ -201,13 +201,15 @@ enum class FrecencyThresholdOption {
  * @property visitTime The time the page was visited in integer milliseconds since the unix epoch.
  * @property visitType What the transition type of the visit is, expressed as [VisitType].
  * @property previewImageUrl The preview image of the page (e.g. the hero image), if available.
+ * @property isRemote Distinguishes visits made on the device and those that come from syncing.
```
Comment (grigoryk) [inline components/concept/storage/src/main/java/mozilla/components/concept/storage/HistoryStorage.kt:204]: nit: "... that come from Sync."

where the name Sync refers to Firefox Sync.

**Review by grigoryk — APPROVED** [review]
Just need to bump app-services version to 91.0.1 which includes the change we need here.

File: components/concept/storage/src/main/java/mozilla/components/concept/storage/HistoryStorage.kt (hunk 1 above)
Comment (mavduevskiy) [reply components/concept/storage/src/main/java/mozilla/components/concept/storage/HistoryStorage.kt]: fixed

**Comment by grigoryk** [comment]
I tested this out locally (the A-S upgrade, specifically), and it appears to be fine. @mavduevskiy did the same on his end.

**Pull request merged**